foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/expressions/Expression.java

```diff
@@ -5447,4 +5450,17 @@ public DatabaseMapping getLeafMapping(DatabaseQuery query, ClassDescriptor rootD
         return null;
     }
 
+    /**
+     * INTERNAL:
+     * Flag to control SQLSelectStatement generation.
+     * true - generate. Usually in case of nested select statement in update query.
+     * false - don't generate (default)
+    */
```

Comment:
keep in mind that javadoc tool will put everything on one line in html, consider adding some formatting tags to keep the javadoc readable; also add the documentation to the corresponding getter

Reply:
Also note the convention is to use _should_/_setShould_ for booleans instead of _is_